.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+---
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: cimg/node:10.21.0
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: npm install
+      - run:
+          name: Build
+          command: npm run build
+      - run:
+          name: Test
+          command: npm run test
+      - run:
+          name: Semantic release
+          when: on_success
```

Comment:
on_success is not needed. Circle doesnt do the next step if the previous one fails